src/meta/src/hummock/manager/compaction/compaction_group_schedule.rs

```diff
@@ -830,15 +836,16 @@ impl HummockManager {
 
     pub async fn try_merge_compaction_group(
         &self,
-        table_write_throughput: &HashMap<u32, VecDeque<u64>>,
+        table_write_throughput_statistic_manager: &TableWriteThroughputStatisticManager,
         group: &CompactionGroupStatistic,
         next_group: &CompactionGroupStatistic,
-        checkpoint_secs: u64,
         created_tables: &HashSet<u32>,
     ) -> Result<()> {
         // TODO: remove this check after refactor group id
-        if group.group_id == StaticCompactionGroupId::StateDefault as u64
-            && next_group.group_id == StaticCompactionGroupId::MaterializedView as u64
+        if (group.group_id == StaticCompactionGroupId::StateDefault as u64
+            && next_group.group_id == StaticCompactionGroupId::MaterializedView as u64)
+            || (group.group_id == StaticCompactionGroupId::MaterializedView as u64
+                && next_group.group_id == StaticCompactionGroupId::StateDefault as u64)
```

Comment:
Is this reachable? I think `group.group_id` is supposed to be smaller than `next_group.group_id`

Reply:
yes, We cannot guarantee the group_id relationship between cg2 and cg3 in the old cluster.